python/lsst/ts/IntegrationTests/aux_tel_prepare_for_onsky.py

```diff
@@ -0,0 +1,41 @@
+# This file is part of ts_IntegrationTests
```

Comment:
Please normalize the file names here. The top-level scripts use auxtel but these scripts are named aux_tel. I prefer the former.

Reply:
Fixed